src/main/java/math/Fraction.java

```diff
@@ -102,8 +102,7 @@ public Fraction subtract(int n) {
      * @return a fraction that represents the result
      */
     public Fraction multiply(int n) {
-        // TODO (Z79CCJ)
-        return null;
+        return new Fraction(numerator * n, denominator * n);
```

Comment:
This is totally wrong! Note that if you multiply both the numerator and the denominator by n, you get a Fraction object that represents the same rational number as the original Fraction object.